Introduction.md

```diff
@@ -715,6 +715,14 @@ Because `WeakHashSet` does not get notified when the referenced object is delete
 the users / owners of `WeakHashSet` are still responsible for deleting the relevant entries from the set.
 Otherwise, WeakHashSet will hold onto `WeakPtrImpl` until `computeSize` is called or rehashing happens.
 
+### WeakHashMap
+
+Like `WeakHashSet`, `WeakHashMap` is  specialized class to map a WeakPtr key with a value.
```

Comment:
Nit - it looks like there's an extra space after "is". Maybe, "…is a specialized class…"?

Reply:
oh, weird. Don't know what happened there but should be "is a specialized class" as you say.